### ISSUE TITLE
How to solve output label delay?

### ISSUE THREAD

**Issue by poor1017** [baidu-research/warp-ctc#157]
Hi 

    I found the output logit have some delay, how to solve this.

    Tensorflow builtin ctc has a parameter 'output_delay_', can I get the similar parameter in this project?

Thanks!
